### ISSUE TITLE
Crash when trying to use it

### ISSUE THREAD

**Issue by belug23** [msg-systems/ui5-testrecorder#25]
Chrome version: Version 78.0.3904.97 (Official Build) (64-bit)
OS: Linux Ubuntu-mate 18.04

Steps to reproduce:
Install the extension from the chrome store
Go on a web page
Click the extension button
The windows will appear, stay white and crashes after a couple of seconds.

Other information:
- I have 40 tabs open
- I'm trying it on a HTTPS site
- If you need more debug information let me know.

**Comment by belug23** [comment]
I have more information about the crashes! Sadly there's a lot of case sensitive errors in the code.

Here a few exemples:
- ui5/resources/sap/ui/core/html.js (Should be HTML.js)
- scripts/popup/view/start.view.js (Should be Start.view.js)

I'll check if I can do a pull request to solve this if you don't have the time.

**Comment by ManuelB** [comment]
@belug23 I expect that the problem is the following:

```
Error handling response: ReferenceError: sOurWindowId is not defined
```

![Screenshot from 2019-12-16 10-43-49](https://user-images.githubusercontent.com/457641/70896540-2a91d080-1ff1-11ea-83b1-cff32d4b361e.png)

To get this do the following:

1. Go to chrome://extensions/
2. Click on Details in the extension UI5 Test-Recorder
3. Enable collect errors (Screenshot below)
4. Start the extension
5. Observe that it crashes
6. Go again to chrome://extensions/
7. Make sure that Developer mode is on (switch on the upper right side)
8. Click in the extension card on Erros

![Screenshot from 2019-12-16 10-46-23](https://user-images.githubusercontent.com/457641/70896626-557c2480-1ff1-11ea-8e2d-52b446a8b3b1.png)

I will try to fix this and send a pull request.

**Comment by martenAdrian** [comment]
Merged #28, the issue should be resolved. 
Can you @belug23 please checkout the developement branch and retry?
